packages/core/test/unit/html/NodeProcessor.test.ts

```diff
@@ -2,11 +2,16 @@ import path from 'path';
 import cheerio from 'cheerio';
 import htmlparser from 'htmlparser2';
 import * as testData from './NodeProcessor.data';
+import * as logger from '../../../src/utils/logger';
 import { Context } from '../../../src/html/Context';
 import { shiftSlotNodeDeeper, transformOldSlotSyntax } from '../../../src/html/vueSlotSyntaxProcessor';
 import { getNewDefaultNodeProcessor } from '../utils/utils';
 import { MbNode, parseHTML } from '../../../src/utils/node';
 
+jest.mock('../../../src/utils/logger', () => ({
+  warn: jest.fn(),
```

Comment:
I am thinking of changing this to `warn: jest.fn(msg => 'warn: ${msg}')` so that we can check the output string as well. What do you think?